### PR TITLE
Remove path local dependency and replace with project version

### PR DESCRIPTION
**Pull request by fbrv**

### CODE REVIEW

File: src/kraken/std/python/pyproject.py [hunk 1]
```diff
@@ -54,6 +55,30 @@ def save(self, path: Path | None = None) -> None:
     def get_poetry_sources(self) -> list[dict[str, Any]]:
         return list(self._poetry_section().setdefault("source", []))
 
+    def _find_dependencies(self, data: Dict[str, Any], version: str) -> None:
+        pattern = re.compile("^.*dependencies$")
```
Comment (NiklasRosenstein) [inline src/kraken/std/python/pyproject.py:59]: Looking at this method I have to say I'm a bit dumbfounded what exactly is going to  happen here. Can you add as a docstring what the expected input data (e.g. what par t of the pyproject) format is here?

File: .changelog/_unreleased.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[[entries]]
+id = "eb334a1d-dfe5-427b-b791-7b5b43be9f1b"
+type = "feature"
+description = "remove local path to depedencies and replace it with current project version"
```
Comment (NiklasRosenstein) [inline .changelog/_unreleased.toml:4]: Can you be a bit more descriptive what the change does? :)

File: src/kraken/std/python/pyproject.py (hunk 1 above)
Comment (fbrv) [reply src/kraken/std/python/pyproject.py]: My bad,  fixed it by adding a docstring, updating the input variable, and modifying the regex. The objective is to identify all the sections where dependencies are defined and replace any relevant local paths with the corresponding project version. Given the different approaches for specifying dependency sections, I chose to traverse the entire Pyproject file. Specifically, I traverse the dictionary representation of the file and treat any key that matches the regular expression ^(dev-)?dependencies$" as a dependency section.

**Comment by cdbrkfxrpt** [comment]
need project-local branch for checks to pass, closing

**Pull request closed without merging**